lib/jekyll_auth/config.rb

```diff
@@ -17,6 +17,14 @@ def self.whitelist
     Regexp.new(whitelist.join('|')) unless whitelist.nil?
   end
 
+  def self.whitelist_teams
+    JekyllAuth.config
+              .select { |k, _| k =~ /^team_\d+$/ }
```

Comment:
I'm not sure looping through the config for a key with regex is the smartest idea, especially when it comes to security. What if down the line we add a `team_specific_bouncer` option (or any other option that begins with`team_`)?